kore/src/Kore/Repl/Data.hs

```diff
@@ -75,6 +79,8 @@ helpText =
     \config [n]              shows the config for node 'n'\
                              \(defaults to current node)\n\
     \leafs                   shows unevaluated or stuck leafs\n\
+    \prec-branch [n]         shows first preceding branch\n\
```

Comment:
Since we explicitly describe the default for the other commands, I would rather have these two commands also say `(defaults to current node)` in the help.